test/cli-doctor-visualize-only.test.js

```diff
@@ -21,7 +21,12 @@ test('clinic doctor --visualize-only - no issues', function (t) {
       'clinic', 'doctor', '--visualize-only', dirpath
     ], function (err, stdout) {
       t.ifError(err)
-      t.strictEqual(stdout, `Generated HTML file is ${dirpath}.html\nYou can use this command to upload it:\nclinic upload ${dirpath}\n`)
+      t.strictEqual(
+        stdout,
+        'To generate the report press: Ctrl + C\n' +
+        `Generated HTML file is ${dirpath}.html\n` +
+          'You can use this command to upload it:\n' +
+          `clinic upload ${dirpath}\n`)
```

Comment:
Can you move all of this to a multiline string with the backticks?

Reply:
It'll then be aligned at the beginning of the line. Or do you want the whole string in a single line?

Reply:
I’m good with the alignment.